snc-library.sh

```diff
@@ -136,7 +136,11 @@ function replace_pull_secret() {
 }
 
 function create_json_description {
-    openshiftInstallerVersion=$(${OPENSHIFT_INSTALL} version)
+    local preset=${1:-snc}
+    openshiftInstallerVersion=""
+    if [ ${preset} == "snc" ]; then
+        openshiftInstallerVersion=$(${OPENSHIFT_INSTALL} version)
+    fi
```

Comment:
Does this mean `crc-bundle-info.json` will contain:
```
"buildInfo": {
    "openshiftInstallerVersion": ""
}
``` ?
Would be better if we could not add it at all :)

Reply:
I already added this logic.